ts/utils/util.py

```diff
@@ -21,3 +26,54 @@ def list_classes_from_module(module, parent_class=None):
         return [c for c in classes if issubclass(c, parent_class)]
 
     return classes
+
+def load_label_mapping(mapping_file_path):
+    """
+    Load a JSON mapping { class ID -> friendly class name }.
+    Used in BaseHandler.
+    """
+    if not os.path.isfile(mapping_file_path):
+        logger.warning('Missing the index_to_name.json file. Inference output will not include class name.')
+        return None
+
+    with open(mapping_file_path) as f:
+        mapping = json.load(f)
+    if not isinstance(mapping, dict):
+        raise Exception('index_to_name mapping should be in "class":"label" json format')
+
+    # Older examples had a different syntax than others. This code accommodates those.
+    if 'object_type_names' in mapping and isinstance(mapping['object_type_names'], list):
+        mapping = {str(k): v for k, v in enumerate(mapping['object_type_names'])}
+        return mapping
+
+    for key, value in mapping.items():
+        new_value = value
+        if isinstance(new_value, list):
+            new_value = value[-1]
+        if not isinstance(new_value, str):
+            raise Exception('labels in index_to_name must be either str or [str]')
+        mapping[key] = new_value
+    return mapping
+
+def map_class_to_label(probs, mapping=None, lbl_classes=None):
```

Comment:
I see that you have modified the json files and made them consistent with respect to formats. Can you please document this somewhere so that some planning to use this util method will know what is expected.

Reply:
Added to the docs in next commit.